src/heretic/r_segs.c

```diff
@@ -675,15 +676,15 @@ void R_StoreWallRange(int start, int stop)
 
         if (worldlow != worldbottom
             || backsector->floorpic != frontsector->floorpic
-            || backsector->lightlevel != frontsector->lightlevel
+            || backsector->rlightlevel != frontsector->rlightlevel
```

Comment:
Required to avoid flickering glitch in E2M7 Hub Area. Should be fixed in crispy-doom aswell?